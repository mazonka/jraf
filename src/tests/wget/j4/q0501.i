command=jw md 0 /.jraf.sys/a : read 0 /.jraf.sys/a : md 0 /.jraf.sys/users : md 0 /.jraf.sys/a