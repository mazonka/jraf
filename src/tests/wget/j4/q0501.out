OK /.jraf.sys/a : OK 0 -1 0 : OK /.jraf.sys/users : JRAF_FAIL system path